tests/behat/features/bootstrap/SocialDrupalContext.php

```diff
@@ -450,4 +451,49 @@ public function iWaitForTheInstallerBatchJobToFinish() {
     $this->getSession()->wait(1800000, 'jQuery("#updateprogress").length === 0');
   }
 
+  /**
+   * Add likes to node at the start of a test with existing users as authors.
+   *
+   * Creates like provided in the form:
+   * | title    | bundle | user      |
+   * | My event | event  | Jane Doe  |
+   * | ...      | ...    | ...       |
+   *
+   * @Given likes node with defined author:
+   */
+  public function createLikesWithDefinedAuthor(TableNode $nodesTable): void {
```

Comment:
@volodymyr-sydor I would propose to rename method to `createLikes`

Reply:
@ribel  I would recommend using with a similar name as for other cases